README.md

```diff
@@ -75,6 +75,33 @@ console.log(prediction.status); // "starting"
 From there, you can fetch the current status of the prediction using
 `await prediction.load()` or `await replicate.prediction(prediction.id).load()`.
 
+You can also provide webhook configuration to have Replicate send POST requests
+to your service when certain events occur:
+
+```js
+import replicate from "replicate";
+
+await replicate
+  .model(
+    "stability-ai/stable-diffusion:db21e45d3f7023abc2a46ee38a23973f6dce16bb082a930b0c49861f96d1e5bf"
```

Comment:
Conjecture warning: This version is SD 2.x, which had artists removed? Or something? So maybe the warhol prompt might not be the most appropriate.